examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { PrismaService } from '../../../../../prisma/prisma.service';
+import { NOT_FOUND_ERR_MESSAGE } from '../api/err.messages';
+import { BpiAccount } from '../models/bpiAccount';
+
+// Repositories are the only places that talk the Prisma language of models.
```

Comment:
@ognjenkurtic Quick question, If this is a prisma repository, why are we naming it as `StorageAgent`, and not just `BpiAccountRepo`

Reply:
I believe we originally used repo and decided to move over to agent to correlate functionality between the two injectable agents.  I also remember a discussion of the word "repo" possibly introducing confusion upon first look at the repository code.